docs/docusaurus.config.js

```diff
@@ -124,8 +111,8 @@ const config = {
             title: 'More',
             items: [
               {
-                label: 'Blog',
-                to: '/blog',
+                label: (process.env.BLOG) ? 'Docs' : 'Blog',
+                href: (process.env.BLOG) ? 'https://docs.corsobackup.io/' : 'https://blog.corsobackup.io/',
```

Comment:
I think this one is revesed

Reply:
I think this is fine. If it is the blog site, we want a link to docs. If it is a docs site, we want a link to the blog. Does that make sense?

Reply:
Ahh, I misread it.